File: 2566/2566.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   2566.cpp                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2023/06/06 16:54:50 by jihoolee          #+#    #+#             */
/*   Updated: 2023/06/06 17:10:32 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>

const int SIZE = 9;

int main(void) {
  int max = 0;
  int max_row = 1, max_col = 1;
  int curr_num;

  for (int row = 0; row < SIZE; ++row) {
    for (int col = 0; col < SIZE; ++col) {
      std::cin >> curr_num;
      if (curr_num > max) {
        max = curr_num;
        max_row = row + 1;
        max_col = col + 1;
      }
    }
  }
  std::cout << max << std::endl << max_row << ' ' << max_col;
  return 0;
}

File: 2566/2566.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    2566.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2023/06/06 17:12:04 by jihoolee          #+#    #+#              #
#    Updated: 2023/06/06 17:41:38 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

SIZE = 9


def main():
    mat = [list(map(int, input().split())) for _ in range(SIZE)]
    max_val = float('-inf')
    max_row = 1
    max_col = 1
    for row_mat in mat:
        max_in_row = max(row_mat)
        if (max_in_row > max_val):
            max_val = max_in_row
            max_row = mat.index(row_mat) + 1
            max_col = row_mat.index(max_val) + 1
    print(max_val)
    print(max_row, max_col)


if (__name__ == '__main__'):
    main()
